crates/topos-tce-storage/src/validator/mod.rs

```diff
@@ -138,6 +138,17 @@ impl ValidatorStore {
             .property_int_value(ESTIMATE_NUM_KEYS)?)
     }
 
+    /// Returns the number of certificates in the pending pool (by iterating)
+    pub fn iter_count_pending_certificates(&self) -> Result<u64, StorageError> {
```

Comment:
Why do you prefer a `u64` over `usize` for the counts here (and below)?

Reply:
To match the type of the other method (estimate key) I'm not really sure about keeping those methods tho.